pkg/apis/pingcap/v1alpha1/backup.go

```diff
@@ -202,6 +202,15 @@ func IsBackupPrepared(backup *Backup) bool {
 	return condition != nil && condition.Status == corev1.ConditionTrue
 }
 
+// IsBackupStopped returns true if a Backup is stopped.
+func IsBackupStopped(backup *Backup) bool {
```

Comment:
What's the difference between `IsBackupStopped` and `IsLogBackupAlreadyStop`? Can we use `IsLogBackupAlreadyStop` to replace `IsBackupStopped`?

Reply:
`IsBackupStopped` is more likely to be in this state, such as command is stop and status is stopped, but command is truncate and status is stopped is not this status, and this is why truncate can run after stopped. `IsLogBackupAlreadyStop` is more likely to don't care about specific subcommands. so they a two function now.

Reply:
Can you add some comments to this function to interpret its usage?

Reply:
ok